train.py

```diff
@@ -110,8 +114,8 @@ def main():
             print("\nRestored coarse model from checkpoint")
             coarse_model.load_weights(latest_checkpoint_coarse)
         else:
-            print("\nCoarse model not restored. Please run coarse model first")
-            return
+            print("\nCoarse model not restored. Please exit and run coarse model first")
```

Comment:
we can merge this if you remove the one pass training part

Reply:
I thought let us keep one pass option in production too. wdyt?